db/seeds.rb

```diff
@@ -2,7 +2,7 @@
 # It should not be specific to a particular instance.
 
 Division.root.destroy if Division.root.present?
-Division.create(id: 99, name: 'Root Division') unless Division.root
+Division.find_or_create_by(id: 99, name: 'Root Division', short_name: 'root-division') unless Division.root
```

Comment:
Good call, except that `find_or_create_by` tests that _all_ things match, and the `Division.root.destroy` makes the whole thing sorta moot anyway. I want us to spend some time gradually refactoring the seeds so that we _can_ safely re-run them though, so I suggest using something like

`Division.find_or_initialize_by(id: 99).assign_attributes(name: 'Root Division', short_name: 'root-division').save!` (dropping the unless was intentional) and then cutting the `Division.root.destroy` line.